docs/antora-aissemble-ui/.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
```

Comment:
I (Informational): you can basically ignore everything in the `antora-aissemble-ui` directory